File: 3172-divisible-and-non-divisible-sums-difference/divisible-and-non-divisible-sums-difference.py
class Solution:
    def differenceOfSums(self, n: int, m: int) -> int:
        dsum = 0
        ndsum = 0
        for i in range(1,n+1):
            if i % m == 0 :
                dsum += i
            else:
                ndsum += i
        return ndsum- dsum

        

File: 3172-divisible-and-non-divisible-sums-difference/divisible-and-non-divisible-sums-difference.js
var differenceOfSums = function(n, m) {
    dsum = 0;
    ndsum = 0;
    for (let i = 1; i <=n; i++){
        if (i % m == 0)
        dsum += i;
        else 
        ndsum +=i;
    }
    return ndsum - dsum;
    
};